cpp/src/ray/ray_config_internal.cc

```diff
@@ -0,0 +1,84 @@
+
+#include "gflags/gflags.h"
+
+#include "ray_config_internal.h"
+
+DEFINE_string(redis_address, "", "The ip address of redis server.");
+
+DEFINE_string(redis_password, "", "The password of redis server.");
```

Comment:
```suggestion
DEFINE_string(redis_password, "5241590000000000", "The password of redis server.");
```

Reply:
@kfstorm This default value is never used.